src/schema/index.ts

```diff
@@ -650,10 +654,29 @@ const Schema = gql`
     sanitizedAttendance: SanitizedAttendance!
   }
 
+
+  type traineeAttendanceWeek {
+    week: Int!
+    weekAverage: String!
+    daysStatus: AttendanceDates!
+  }
+
+  type traineeAttendancePhase {
+    phase: Phase!
+    phaseAverage: String!
+    weeks: [traineeAttendanceWeek]!
+  }
+
+  type traineeAttendance {
+    traineeId: String!
+    teamName: String!
+    allPhasesAverage: String!
+    phases: [traineeAttendancePhase]!
+  }
```

Comment:
Is it necessary to break these into separate types?

Reply:
I think this should go to the team that has been working on attendance as i never altered any of these types and i think if i do alter or combine these types it might affect their work. Thanks